volontulo/templates/homepage.html

```diff
@@ -23,22 +23,38 @@
 
     {% if offers %}
         <h2>Lista ofert</h2>
-        <table class="table table-striped table-bordered">
-            <tr>
-                <th>Tytuł</th>
-                <th>Miejsce</th>
-                <th>Nazwa organizacji</th>
-                <th>Czas obowiązywania</th>
-            </tr>
+        <div class="row offer-thumbnails">
         {% for o in offers %}
-            <tr>
-                <td><a href="{% url 'offers_view' o.title|slugify o.id %}">{{ o.title }}</a></td>
-                <td>{{ o.location }}</td>
-                <td><a href="{% url 'organization_view' slug=o.organization.name|slugify id_=o.organization.id %}">{{ o.organization.name }}</a></td>
-                <td>{{ o.time_period }}</td>
-            </tr>
+            <div class="col-sm-6 col-md-3">
+                <div class="thumbnail">
+                    <div class="heading-image" style="background-image:url(http://placehold.it/300x190?text=fota+mobilizująca)"></div>
```

Comment:
My suggestion is not using important images as inline styles - it should be img with properly alt attribute defined.